indexing-service/src/test/java/io/druid/indexing/overlord/TaskLockboxTest.java

```diff
@@ -75,10 +80,11 @@ public void testLockForInactiveTask() throws InterruptedException
     lockbox.lock(NoopTask.create(), new Interval("2015-01-01/2015-01-02"));
   }
 
-  @Test(expected = IllegalStateException.class)
+  @Test
   public void testLockAfterTaskComplete() throws InterruptedException
   {
     Task task = NoopTask.create();
+    exception.expect(IllegalStateException.class);
```

Comment:
This should be ISE. Also, it would be better to check the exception message as well. Please refer to AppenderatorDriverFailTest as an example.